nemo/collections/common/tokenizers/text_to_speech/tts_tokenizers.py

```diff
@@ -694,10 +723,7 @@ def __init__(
         pad_with_space=False,
         text_preprocessing_func=chinese_text_preprocessing,
     ):
-        """
-        Chinese phoneme-based tokenizer.
-        Note: This tokenizer for now covers Chinese phonemes/tones and English letters because our dataset contains
-              both Chinese and English graphemes.
+        """Chinese phoneme-based tokenizer.
```

Comment:
just to confirm: is this removal intentional?

Reply:
it was my mistake, thank you for noticing it!